libs/api/domains/documents/tsconfig.json

```diff
@@ -5,5 +5,6 @@
     "target": "es6"
   },
   "allowImplicitAny": false,
+  "strict": true,
```

Comment:
This ok?

Reply:
should be, document domain does not depend on any other parts of the repo so it builds fine in strict mode